Dashboard/app/js/lib/views/data/question-answer-view.js

```diff
@@ -156,6 +162,46 @@ FLOW.QuestionAnswerView = Ember.View.extend({
     return null;
   }.property('this.content'),
 
+  /*
+   * parse the strip test JSON result
+   */
+  parseStriptestJson: function(){
+	var c = this.content, signatureJson, newResult, image;
+	result=Ember.A();
+	if (c && c.get('value')) {
+	  striptestJson = JSON.parse(c.get('value'));
+	  if (striptestJson.result && !Ember.empty(striptestJson.result)){
+		  striptestJson.result.forEach(function(item){
+			  image = 'data:image/png;base64,' + item.img;
```

Comment:
I don't think so, as the forEach function iterates over the result array from the JSON, which contains the different images as values. The image is created using item.img.